app/models/activity_insight_oa_file.rb

```diff
@@ -21,6 +21,14 @@ class ActivityInsightOAFile < ApplicationRecord
       .where(downloaded: nil)
       .where.not(location: nil)
   }
+  scope :send_oa_status_to_activity_insight, -> {
+                                               left_outer_joins(:publication)
+                                                 .where(publication: { publication_type: Publication.oa_publication_types })
+                                                 .left_outer_joins(publication: :open_access_locations)
+                                                 .where(open_access_locations: { publication_id: nil })
+                                                 .where(%{publication.open_access_status = 'gold' OR publication.open_access_status = 'hybrid'})
```

Comment:
I think we also want to send OA status if there's a ScholarSphere open access location.